events/code_review_events/workflow.py

```diff
@@ -219,7 +225,44 @@ def publish_results(self, payload):
         return True
 
     def parse_pulse(self, payload):
-        assert self.publish is True, "Publication disabled"
+
+        routing = payload["routing"]
+
+        # Process autoland payloads
+        if routing["exchange"] == PULSE_TASK_GROUP_RESOLVED:
+            try:
+                self.trigger_autoland(payload["body"])
```

Comment:
It'd be good to have an option to disable it (at least at first, then re-enable it once tried and tested)

Reply:
Done